chicken-sloths/bangazon-cli-sprint2
# A few tests for completeOrderView

# Description
Provides tests for `completeOrderV`

## Related Ticket(s)
#6

## Problem to Solve
While we aren't normally testing prompts, this prompt is dynamically generated based upon the active customer, their payment options, and whether they have an active order. Blindly hoping that the prompt for `6` was generated correctly seemed imprudent.

## Proposed Changes
Removes commented out code in `completeOrderV` so it will function normally.
These tests check whether or not the prompt looks as it should.

## Expected Behavior
Selecting `6` from the main menu should exit the process
All test should pass

## Steps to Test Solution

1. `npm test` (all tests should pass)
1. `npm start`
1. Select `6`
1. Application should exit

## Testing
